utils/useLocalStorage.ts

```diff
@@ -7,7 +7,7 @@ import { useState } from "react";
  * @returns [storedValue, setValue]
  * Source: https://usehooks.com/useLocalStorage/
  */
-export default function useLocalStorage(key, initialValue) {
+export default function useLocalStorage(key, initialValue: any) {
```

Comment:
We can type this, this could help you catch some bugs as well:
```suggestion
export default function useLocalStorage<Type>(
    key: string,
    initialValue: Type,
): [Type, (value: Type) => void] {
```